### ISSUE TITLE
Unhandled rejection TypeError: Cannot read property 'nodeName' of null

### ISSUE THREAD

**Issue by airboss001** [MaximBalaganskiy/aurelia-inputmask#1]
Getting the noted error in inputmask-attribute on the line indicated by -->
Is it expecting an additional element?

```
valueChanged() {
		if (!this.input) {
			return;
		}
		if (this.ignoreChange) {
			this.ignoreChange = false;
			return;
		}
		if (this.input.value !== this.value) {
			this.input.value = this.value;
		}
		const label = this.input.nextElementSibling as HTMLLabelElement;
-->		if (label.nodeName === "LABEL") {
			label.classList.add(this.value ? "active" : "inactive");
		}
	}
```

Here is an example of how I am using it:
```
<div class="textareasize">
          <input inputmode="numeric" pattern="^[0-9]*$" class="smalltextinput" value.bind="radial"
            inputmask data-inputmask-regex="^\d{3,3}$"> [ex: 030, 281]
</div>
```

**Comment by MaximBalaganskiy** [comment]
Yes, this is a leftover from my materialize specific code. Needs to be removed. In the meantime the workaround is to add an invisible label. Or just ignore the errors...

**Comment by airboss001** [comment]
Ok, thanks for the update. Easy work around. Really appreciate the share.

**Comment by MaximBalaganskiy** [comment]
Fixed via 1e20275fbfeb4288f0c44389fd40736c0803b5dd
